lib/python/rose/macro.py

```diff
@@ -578,6 +579,7 @@ def load_meta_macro_modules(meta_files, module_prefix=None):
         except Exception:
             rose.reporter.Reporter()(
                 MacroLoadError(meta_file, traceback.format_exc()))
+        sys.path.pop()
```

Comment:
Yes, I think it's fine - anything uncaught (a `BaseException`) would kill everything anyway.